### PR TITLE
clientAvatar.js

### PR DESCRIPTION
**Pull request by LukeBruh**
## Type
- [ ] Bug Fix
- [X] Functions: \$clientAvatar[dynamic (boolean;size (number)]
- [ ] Callbacks: \<Callback Name>
- [ ] Handlers: \<Handler Type>
- [ ] Others: \______

Dependencies (Third Party Modules) needed: <Name | Github Link> (Maximum: 20MB~ size)

Want a credit? Discord tag or other social media link: Luey#0028

Referenced Issue:

## Description
Shows the client's avatar, I ask you to check if the code is right...

### CODE REVIEW

**Comment by XzFirzal** [comment]
Please set the base as `dev` next time

File: package/functions/funcs/clientAvatar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const clientAvatar = async (d) => {
+  let code = d.command.code;
+
+  const inside = d.unpack();
+  if (!inside.splits[0]) {
+    inside.splits.shift();
+    inside.splits.unshift(d.client.user.id);
```
Comment (XzFirzal) [inline package/functions/funcs/clientAvatar.js:7]: What this thing do?

**Comment by DiscordBond** [comment]
$client[avatar] and $userAvatar[$clientID] both exist

File: package/functions/funcs/clientAvatar.js (hunk 1 above)
Comment (USERSATOSHI) [reply package/functions/funcs/clientAvatar.js:7]: nice question

Comment (Kino7916) [reply package/functions/funcs/clientAvatar.js:7]: That if guard is not needed nor required.

**Comment by Kino7916** [comment]
Closed PR for Inactivity, as it is also possible to do with existing functions.

**Pull request closed without merging**